### ISSUE TITLE
GCBM exe crash

### ISSUE THREAD

**Issue by B-Ron12** [cat-miti/gcbm_preprocessing#17]
![image](https://user-images.githubusercontent.com/7143833/29144256-1e1ef690-7d0d-11e7-8d26-34370f837b11.png)

**Comment by naddison1** [comment]
Do you have a screenshot of the error? The order shouldn't matter

**Comment by B-Ron12** [comment]
![image](https://user-images.githubusercontent.com/7143833/29147098-cf1ecacc-7d19-11e7-84d4-9f1233eee6b7.png)

Very descriptive

**Comment by B-Ron12** [comment]
this might be related to this error:

Error getting growth curve for query: select growth_curve_id from (select gccv.growth_curve_id as growth_curve_id, sum(case when c.name in ('AU','LdSpp') then case when cv.value like '?' then 1 when c.name like 'AU' and cv.value like '0' then 4 when c.name like 'LdSpp' and cv.value like '' then 4  else -1000 end else -1000 end) as score from growth_curve_classifier_value gccv inner join classifier_value cv on gccv.classifier_value_id = cv.id inner join classifier c on cv.classifier_id = c.id group by gccv.growth_curve_id) as growth_curve_ranking where score > 0 order by score desc limit 1

**Comment by benhudson85** [comment]
I also experienced a moja.cli.exe crash.  For my TSA, it appears to happen after repeated incidents of trying to match AU and LdSpp that don't exist. They don't exist in my TSA and they don't exist in the province. moja.cli.exe is also working on tiles that are way outside of my TSA area - looking at the tile IDs they appear to cover the entire province

I tried running with a moja.cli.exe that had been copied locally and experienced the same crash.  The only successful run I had was to run from the original configs (...\TSA_31_Soo\04_GCBM\00_configs\SCEN_Base) using the cmd: [path to moja.cli.exe] --config [path to config json] --config_provider [path to config provider json]

I am copying the results of that to the network but it is slow and will apparently take 3 hrs for about 120 MB... Will update when it's copied.
